org-code-javabuilder/media/src/main/java/org/code/media/FontHelper.java

```diff
@@ -0,0 +1,54 @@
+package org.code.media;
+
+import java.awt.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import org.code.protocol.InternalErrorKey;
+import org.code.protocol.InternalServerRuntimeError;
+
+public class FontHelper {
+  private final Map<String, java.awt.Font> fontMap;
+
+  public FontHelper() {
+    this.fontMap = new HashMap<>();
+    this.populateFontMap();
+  }
+
+  public java.awt.Font getFont(Font font, FontStyle fontStyle) {
+    return this.fontMap.get(
+        fontToFilenamePrefix.get(font) + fontStyleToFilenameSuffix.get(fontStyle));
+  }
+
+  private void populateFontMap() {
+    try {
+      for (Font font : Font.values()) {
+        for (FontStyle fontStyle : FontStyle.values()) {
+          String filename =
+              fontToFilenamePrefix.get(font) + fontStyleToFilenameSuffix.get(fontStyle);
```

Comment:
nit: separate method for `String getFileName(Font font, FontStyle fontStyle)` since it's used in a few places?

Reply:
done!